charts/ingress-nginx/tests/admission-webhooks/job-patch/job-patchWebhook_test.yaml

```diff
@@ -76,3 +76,46 @@ tests:
                           fieldRef:
                             apiVersion: v1
                             fieldPath: metadata.namespace
+
+  - it: should create a Job with resizePolicy if `controller.admissionWebhooks.patchWebhookJob.resizePolicy` is set and Kubernetes version >= 1.30
+    capabilities:
+      majorVersion: 1
+      minorVersion: 30
```

Comment:
Have you checked if this is required? I am actually not 100% sure, I can remember we needed that in some other test, but I can barely remember without checking.